### PR TITLE
Update docs for v3.5.0-beta & 2.9.0

### PR DESCRIPTION
**Pull request by ryanRfox**
Docs update for:
- go-algorand https://github.com/algorand/go-algorand/releases/tag/v3.5.0-beta
- indexer https://github.com/algorand/indexer/releases/tag/2.9.0

### CODE REVIEW

**Review by JasonWeathersby — DISMISSED** [review]

File: docs/rest-apis/algod/v2.md [hunk 1]
```diff
@@ -841,7 +936,7 @@ POST /v2/teal/compile
 
 
 **Description**
-Given TEAL source code in plain text, return base64 encoded program bytes and base32 SHA512_256 hash of program bytes (Address style). This endpoint is only enabled when a node's configuration file sets EnableDeveloperAPI to true.
+Given TEAL source code in plain text, return base64 encoded program bytes and base32 SHA512_256 hash of program bytes (Address style). This endpoint is only enabled when a node's configureation file sets EnableDeveloperAPI to true.
```
Comment (barnjamin) [inline docs/rest-apis/algod/v2.md:939]: ???

File: docs/rest-apis/algod/v2.md [hunk 2]
```diff
@@ -890,7 +985,7 @@ POST /v2/teal/dryrun
 
 
 **Description**
-Executes TEAL program(s) in context and returns debugging information about the execution. This endpoint is only enabled when a node's configuration file sets EnableDeveloperAPI to true.
+Executes TEAL program(s) in context and returns debugging information about the execution. This endpoint is only enabled when a node's configureation file sets EnableDeveloperAPI to true.
```
Comment (barnjamin) [inline docs/rest-apis/algod/v2.md:988]: here too?

Comment (ryanRfox) [reply docs/rest-apis/algod/v2.md]: TODO: fix this in go-algorand source

**Comment by nullun** [comment]
I've just realised the typo was introduced due to a change I merged last week. I've PR'ed to go-algorand since that's where it originates. https://github.com/algorand/go-algorand/pull/3741/files

File: docs/rest-apis/algod/v2.md [hunk 3]
```diff
@@ -133,15 +133,16 @@ Given a specific account public key, this call returns the accounts status, bala
...
 
 
 **Responses**
 
 |HTTP Code|Description|Schema|
 |---|---|---|
-|**200**||[Account](#account)|
-|**400**|Malformed address|[ErrorResponse](#errorresponse)|
+|**200**|AccountResponse wraps the Account type in a response.|[Account](#account)|
+|**400**|Bad request|[AccountsErrorResponse](#accountserrorresponse)|
```
Comment (barnjamin) [inline docs/rest-apis/algod/v2.md:145]: I _think_ this is going to change with https://github.com/algorand/go-algorand/pull/3737

File: docs/rest-apis/algod/v2.md (hunk 2 above)
Comment (cce) [reply docs/rest-apis/algod/v2.md]: Merged @nullun's https://github.com/algorand/go-algorand/pull/3741

**Comment by barnjamin** [comment]
This was superseded by PRs from eng team, closing

**Pull request closed without merging**